### PR TITLE
Fixed out of bounds and image for poke-evolution-component

### PR DESCRIPTION
**Pull request by philbu**
See title. 
#114

### CODE REVIEW

File: ionic2/app/components/evolutions/evolutions.component.scss [hunk 1]
```diff
@@ -10,8 +10,10 @@
 }
 
 .evolutions > div > button {
-  width: 100px;
-  height: 100px;
+  max-width: 100px;
+  max-height: 100px;
+  width: 23vw;
+  height: 23vw;
```
Comment (MajorBreakfast) [inline ionic2/app/components/evolutions/evolutions.component.scss:16]: IMO percentages are better than viewport width & height. That way the sizes of child elements are relative to the component's size. Makes for better encapsulation.

Comment (philbu) [reply ionic2/app/components/evolutions/evolutions.component.scss:16]: You're right but in this case it's better/easier to go with vw.

Comment (philbu) [reply ionic2/app/components/evolutions/evolutions.component.scss:16]: Are you okay with that solution?

**Review by mfkaptan — APPROVED** [review]

**Comment by johartl** [comment]
@philbu Can this branch be deleted?

**Pull request merged**